tests/lifecycle/parameters/parameters.go

```diff
@@ -48,4 +49,7 @@ var (
 		testPodLabelPrefixName: testPodLabelValue,
 		"app":                  "test",
 	}
+	AffinityRequired = map[string]string{
```

Comment:
Minor suggestion: I'll add the suffix "PodLabels" to this var name: `AffinityRequiredPodLabels`.